packages/components/tests/dummy/app/templates/components/form/radio.hbs

```diff
@@ -0,0 +1,585 @@
+  <dl class="dummy-component-props" aria-labelledby="component-api-form-radio-group">
+    <dt>layout <code>enum</code></dt>
+    <dd>
+      <p>
+        Sets the layout of group.
+      </p>
+
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">vertical</li>
+        <li>horizontal</li>
+      </ol>
+    </dd>
+  </dl>
+  <h5 class="dummy-h5">Contextual components</h5>
+  <p class="dummy-paragraph" id="component-api-form-radio-group-contextual-components">Legend, group of fields and error
+    content are passed to the group as yielded components, using the
+    <code class="dummy-code">Legend</code>,
+    <code class="dummy-code">Radio::Field</code>,
+    <code class="dummy-code">Error</code>
+    keys.</p>
+  <p class="dummy-paragraph"><em>Notice: the group of elements is automatically wrapped in a
+      <code class="dummy-code">&lt;fieldset&gt;</code>
+      element.</em></p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-form-radio-group-contextual-components">
+    <dt>&lt;[C].Legend&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is an (optional) container that yields its content inside the
+        <code class="dummy-code">&lt;legend&gt;</code>
+        element.</p>
+      <p>The content can be a simple string, or a more complex/structured one (in which case it inherits the text
+        style).</p>
+      <p>For details about its API check the
+        <LinkTo @route="components.form.base-elements"><code class="dummy-code">Form::Legend</code></LinkTo>
+        component.</p>
+    </dd>
+    <dt>&lt;[C].Radio::Field&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is used to yield one or more fields inside the group.</p>
+      <p>For details about its API check the
+        <code class="dummy-code">Radio::Field</code>
+        component above.</p>
+    </dd>
+    <dt>&lt;[F].HelperText&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content inside the "helper text" block (at group level).</p>
+      <p>The content can be a simple string, or a more complex/structured one (in which case it inherits the text
+        style).</p>
+      <p>For details about its API check the
+        <LinkTo @route="components.form.base-elements"><code class="dummy-code">Form::HelperText</code></LinkTo>
+        component.</p>
+      <p><em>Notice: the
+          <code class="dummy-code">id</code>
+          attribute of the element is automatically generated.</em></p>
+    </dd>
+    <dt>&lt;[F].Control&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a very simple container that yields its content. It is used to forward the "field" control inside the
+        fields' "group" control wrapper.</p>
+      <p><em>Notice: you can pass all the controls to a single
+          <code class="dummy-code">&lt;Control&gt;</code>
+          container, or you can have one control per container.</em></p>
+    </dd>
```

Comment:
One of them is at the `::Field` level and the other at the `::Group` level that's why we expose them both. I'd argue it's important to keep them otherwise people may not know that you can use let's say helper text at any of the two levels.

Reply:
I am not sure why I misunderstood the code (I think because of the `[F].Control` which should have not been there and the `F.` prefix). In any case, reading the docs now they are correct, so all good 👍